src/components/button-group/button-group.scss

```diff
@@ -14,6 +14,8 @@
     grid-auto-flow: column;
     background-color: transparent;
     margin: functions.pxToRem(2) !important;
+    font-size: functions.pxToRem(14);
+    line-height: 1.25rem;
```

Comment:
Here, the line-height does not seem to have any visual effect (because of `align-items: center;`) on `.mdc-chip`, neither for iconed buttons, nor for text-only buttons. Thus can be removed.


https://user-images.githubusercontent.com/35954987/125753224-b1001440-e510-44e4-9969-93379644de3f.mp4

Reply:
While we should certainly strive to clean stuff like this up, this is things I don't think we should spend _any_ time on before we get this merged.

Reply:
Right, but it may be hard to forget it in future. Since we will never remember when/why it was added, and whether it can be removed or not. Usually these will remain in the code, and in some other situations we will have to override them elsewhere without knowing that they weren't needed. :) So I'll remove this with a fixup for now. If I find new things like this, I suggest we remove them too.

Reply:
Ok. But since it's now reported in this comment, it at least won't be forgotten as long as this PR is open. So can you do the slider styling first? 😁

Reply:
😄 for now the input-field is killing me. After that I'll prioritize the slider ;)